main/src/library/Set.flix

```diff
@@ -30,6 +30,7 @@ namespace Set {
     ///
     /// Returns the size of `xs`.
     ///
+    @Time(size(xs)) @Space(Int32.log2(size(xs)))
```

Comment:
I mean, is this not practically O(1) space for all purposes?

Reply:
That is a fair point.